low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server_manifest.py

```diff
@@ -0,0 +1,335 @@
+#!/bin/python3
```

Comment:
Move all the server related things into lr2/server/, e.g. lr2/server/manifest.py. Similarly there should be a folder lr2/storage/ containing storage interfaces.

Reply:
Actually, common infrastructure code is available in https://github.com/satish-seagate/cortx-monitor/tree/eos-17458-CortxV2-SSPL-Enhance-System-Manifest/low-level/framework/platforms

Here we just interface implementation code. so that's why we have kept it in this way.

Reply:
Ujjwal is mentioning about the structure.

lr2/server/health.py
lr2/server/manifest.py

lr2/storage/health.py
lr2/storage/manifest.py

Reply:
Yes @mariyappanp, I thought he is talking about common infrastructure code which we did the same way as you have explained. Do you need the same structure code here too? Here we just interface implementation code. so that's why we have kept it in this way.

Reply:
Yes, for ease of maintenance.

Reply:
Done